bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -321,6 +321,9 @@ class BCStateTran : public IStateTransfer {
     SendingBatch,
   };
 
+ public:
```

Comment:
Please remove the public, should be protected, and use a delegator in any UT.

Reply:
Thanks. Addressed.